paddle/fluid/framework/operator.cc

```diff
@@ -64,9 +64,9 @@ static DDim GetDims(const Scope& scope, const std::string& name,
 
   if (var->IsType<LoDTensor>()) {
     const LoDTensor& tensor = var->Get<LoDTensor>();
-    if (UNLIKELY(!tensor.IsInitialized())) {
-      return DDim({-1});
-    }
+    // if (UNLIKELY(!tensor.IsInitialized())) {
```

Comment:
why change this, seems not safe?

Reply:
> why change this, seems not safe?

this part to code is just used to print debug message, since now we may have "Shape Variable" which only used to pass the shape info between forward and backward op with uninitialized tensor, remove the code above to show the shape info of the Shape Variable